src/openproduct/utils/pagination.py

```diff
@@ -1,7 +1,20 @@
+from django.utils.translation import gettext as _
+
 from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
 
 
 class Pagination(PageNumberPagination):
     page_size = 100
     page_size_query_param = "page_size"
     max_page_size = 200
+
+    def get_paginated_response(self, data):
+        return Response(
+            {
+                _("aantal"): self.page.paginator.count,
+                _("volgende"): self.get_next_link(),
+                _("vorige"): self.get_previous_link(),
+                _("resultaten"): data,
```

Comment:
This means that the keys of the API response are translatable, so if translations are defined for the strings at some point and you pass an `Accept-Language` header, it will be translated:

![afbeelding](https://github.com/user-attachments/assets/c6a28e20-23b9-4a93-9d85-8ffdca842e38)

![afbeelding](https://github.com/user-attachments/assets/eb3cc88a-55b6-40a4-ab1a-cdba8733b9ca)

I don't think this is desirable? I think the keys of the API responses should be fixed. It also seems to violate this design rule: https://gitdocumentatie.logius.nl/publicatie/api/adr/2.0.2/#/core/interface-language

Reply:
Removed gettext stuff